components/kubeflow/kfserving/component.yaml

```diff
@@ -16,6 +16,8 @@ inputs:
   - {name: Enable Istio Sidecar,            type: Bool,   default: 'True',     description: 'Whether to enable istio sidecar injection'}
   - {name: InferenceService YAML,           type: String, default: '{}',         description: 'Raw InferenceService serialized YAML for deployment'}
   - {name: Watch Timeout,                   type: String, default: '120', description: "Timeout seconds for watching until InferenceService becomes ready."}
+  - {name: Min Replicas,                    type: String, default: '0', description: 'Minimum number of InferenceService replicas'}
```

Comment:
So I checked with the Kubeflow 1.2 release (KFServing 0.4.1), the default min replicas is 1. Can we change the default value here or make it pass None to the spec?
https://github.com/kubeflow/kfserving/blob/release-0.4/pkg/constants/constants.go#L72

Reply:
Committed code to set it to None in case its value is less than 1. This would force it to use the default value.

Reply:
If you set the min_replicas value to None when value is less than 1, then you no longer can make it scale from 0. How about you revert your last commit and make the min/max replicas default values to `-1` to pass None?

Reply:
Yeah that makes sense. I have committed the suggested changes